paimon-core/src/main/java/org/apache/paimon/table/TableUtils.java

```diff
@@ -66,4 +81,51 @@ public static void deleteWhere(Table table, List<Predicate> filters, Lock.Factor
             throw new RuntimeException(e);
         }
     }
+
+    public static FileIO tableFileIO(CatalogContext context) {
+        FileIO fileIO;
+        try {
+            fileIO = FileIO.get(CoreOptions.path(context.options()), context);
```

Comment:
This is not the recommended way, stuffing path into catalogoption is not a good idea, we should avoid it.
Remove all utils here.

Reply:
done.